### ISSUE TITLE
[BUG] MacOS+Qt5: Blurred image in the main canvas

### ISSUE THREAD

**Issue by Kazzz-S** [klayoutmatthias/klayout#55]
About Mac:
 1. OS: Yosemite (10.10.5)
 2. Model: Macbook Pro (13-inch, Mid 2012)
 3. Memory: 16GB
 4. Graphics: Intel HD Graphics 4000 1024MB

**1. KLayout 0.24.10 + Qt 4.8.7 + Yosemite (real machine):**
![klayout-02410](https://user-images.githubusercontent.com/16847255/34472641-bb90aa86-efa9-11e7-8b5b-ffa3e3fb5fde.png)

**2. KLayout 0.25 +Qt 5.10.0 + Yosemite (real machine):**  Partially transparent to the backgroud
![klayout-025](https://user-images.githubusercontent.com/16847255/34472615-22cb8884-efa9-11e7-90e9-b66cee6202c9.png)

**3. KLayout 0.25 + Qt 5.9.3 + High Sierra(virtual machine)**
![virtualhighsierra qt593](https://user-images.githubusercontent.com/16847255/34510877-dd24012e-f09a-11e7-99c0-e31f1a3d96e4.png)

**4. KLayout 0.25 + Qt 5.10.0 + High Sierra(virtual machine)**
![virtualhighsierra qt510](https://user-images.githubusercontent.com/16847255/34512206-26aeabf6-f0a4-11e7-8c70-b4826080c7d1.png)

**5. KLayout 0.25 + Qt 5.9.3 + Yosemite(real machine)**
![yosemite qt593](https://user-images.githubusercontent.com/16847255/34513263-d1620fd8-f0aa-11e7-8e59-a8d57a9d257c.png)

**Comment by Kazzz-S** [comment]
I've added three more different combinations as above.
It looks this is a problem in combination with Qt 5.10.0.

Qt 5.9.3 is fine.
I'm going to use Qt 5.9.3 only.

**Comment by Kazzz-S** [comment]
@klayoutmatthias 

Luckily, this is not a blocker.

Kazzz

**Comment by thomaslima** [comment]
Dear @Kazzz-S . 

I believe this was fixed in #89 . I have no issues with qt5.10 from brew. Try building from [this commit](https://github.com/lightwave-lab/klayout/commit/fa96f6d35232e81134de467bc04b96931ab644f6#diff-9c5dcb7b173366d2e4a62fa1f450a339).

**Comment by Kazzz-S** [comment]
Dear @thomaslima,

Thank you for your effort.
I'll try ;-)

Kazzz

**Comment by Kazzz-S** [comment]
Hi @thomaslima,

I've merged dfd5bb1381a747f582c3dc6ab89df40e0a7aa4a4 and tried to build it on Yosemite.

```
$ brew install python3 qt  ===> Success (Qt is 5.10.1 now)

$ ./build4mac.py -p B36 -q Qt5Brew  ==> Success

$ ./build4mac.py -p B36 -q Qt5Brew -Y ==> Success
```
However ...
```
$ pwd
$HOME/GitWork/klayout-0.25/qt5.pkg.macos-Yosemite-release/klayout.app/Contents/MacOS

$ ./klayout -e
```
**ERROR: Got empty string where a real number was expected**
The same message appears in a GUI dialog on start-up and the program terminates
when [OK] button is pressed.
The default build (-p sys -r sys -q qt5macports) shows the same error.

Have you experienenced the same error?
I'm planning to repeat this on "High Sierra", too.

Regards,
Kazzz

**Comment by thomaslima** [comment]
Yes I think I got that error and posted in #85. But it was fixed by Matthias. Try starting layout with a fresh .klayout

**Comment by Kazzz-S** [comment]
Hi @thomaslima,

Thank you for feedback. I've overlooked the post #85. 
With a fresh ./klayout/, I could start the tool normally.

I've also confirmed that the original issue captured in this ticket was fixed as below.
KLayout 0.25.2 is working fine with Qt 5.10.x.

Qt\OS     | Yosemite |High Sierra|
-----------|-----------|--------------
5.10.0 MacPort|  OK  | OK  
5.10.1 Home Brew|  OK  |  OK  

Thank you once again for your effort.
I'm going to modify the relevant files.

Regards,
Kazzz
